src/newItinerary/internals/Line.style.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, radius, space } from '../../_utils/branding'
+import { ITINERARY_ITEM_BASE_HEIGHT } from '../Itinerary.style'
+import { Lines } from '../Lines'
+
+const LINE_WIDTH = '4px'
+const LINE_HEIGHT = '12px' // First line is 12px long, the second is flex.
+
+export const StyledLineWrapper = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: ${ITINERARY_ITEM_BASE_HEIGHT}px;
+  margin: 0 ${space.m};
+  width: ${space.m};
+`
+
+// Using background-image to display the dashed line for connection
+const base64EncodedDash =
+  'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMCIgaGVpZ2h0PSIxMDAiPjxsaW5lIHgxPSIyIiB4Mj0iMiIgeTE9IjUiIHkyPSIxMDAlIiBzdHJva2U9IiM3MDhDOTEiIHN0cm9rZS13aWR0aD0iMyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtZGFzaGFycmF5PSIuMDAxLCAxMCIvPjwvc3ZnPg=='
+
+const lineToBackground: Readonly<{ [key in Lines]: string }> = {
+  [Lines.NONE]: 'transparent',
+  [Lines.ACTIVE]: color.midnightGreen,
+  [Lines.INACTIVE]: color.gray,
+  [Lines.CONNECTION]: `repeat-y url("data:image/svg+xml;base64,${base64EncodedDash}")`,
+  [Lines.HIDDEN_STOPS]: color.midnightGreen,
+}
+
+export const StyledLine = styled.div<{ line: Lines }>`
+  width: ${LINE_WIDTH};
+  height: calc(${LINE_HEIGHT} + 1px); // Adding 1 magic px to have the line go under the bullet
```

Comment:
Without the margin:
<img width="224" alt="Capture d’écran 2021-02-15 à 18 37 41" src="https://user-images.githubusercontent.com/729186/107978499-27abba80-6fbd-11eb-9107-06f757a81dd7.png">

I didn't update the LINE_HEIGHT constant as it's used in various computations (that work as it)